spec/views/post_index_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe 'User Profile', type: :feature do
+  before do
+    # Create a user and their associated posts and comments for testing
```

Comment:
- [ ] Please as requested by the previous reviewer, do not leave unused code as a comment in your codebase. this will make your code much noisier, It is always a good practice to keep your code clean from such comments. 😉 👍🏻 

- [ ] As requested by the previous reviouwer, could you address these remaining featured tests for the User post index page? 👇🏻 



![image](https://github.com/Zeph180/blog-it/assets/79058364/2ffb8450-0668-44c4-82fb-53d5b3091ca0)